spilled_queries.sql

```diff
@@ -44,6 +45,7 @@ SELECT
     USER_NAME, 
     C.WAREHOUSE_NAME,
     C.WAREHOUSE_SIZE,
+    MAX(C.START_TIME) MOST_RECENT_START_TIME,
```

Comment:
Could you add an 'AS' and change this to: 
```sql
MAX(C.START_TIME) AS MOST_RECENT_START_TIME,
```

Reply:
done

Reply:
👍👍